Toggl.Foundation.MvvmCross/ViewModels/SelectCountryViewModel.cs

```diff
@@ -57,7 +52,8 @@ public override async Task Initialize()
                 allCountries.Insert(0, selectedElement);
             }
 
-            Countries = filterText
+            Countries = FilterText
+                .Select(text => text ?? string.Empty)
                 .Select(text => text.Trim())
```

Comment:
You can save one line of code:
```suggestion
                 .Select(text => text?.Trim() ?? string.Empty)
```